server/src/main/java/org/elasticsearch/search/aggregations/AggregationPhase.java

```diff
@@ -125,7 +125,7 @@ public void execute(SearchContext context) {
         for (Aggregator aggregator : context.aggregations().aggregators()) {
             try {
                 aggregator.postCollection();
-                aggregations.add(aggregator.buildAggregation(0));
+                aggregations.add(aggregator.buildAggregations(new long[] {0})[0]);
```

Comment:
Can we use `aggregator.buildTopLevel()` ?

Reply:
++